content/docs/goFSH/running/_index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Running GoFSH"
+weight: 10
+---
+
+## Running GoFSH
+
+GoFSH is executed from the command line. The general form of the GoFSH execution command is as follows:
+
+```shell
+{{< terminal >}} gofsh {input-directory} {options}
+```
+
+where options include the following (in any order):
+
+```text
+-o, --out <out>                   the path to the output directory (default: ./fsh)
+-l, --log-level <level>           specify the level of log messages: error, warn, info (default), debug
+-d, --dependency <dependency...>  specify dependencies to be loaded using format dependencyId@version (FHIR R4 included by default)
+-v, --version                     print goFSH version
+-h, --help                        output usage information
+```
+
+While GoFSH is running, it will print status messages as it processes your project files.
+
+## GoFSH Inputs
+
+As input, GoFSH takes FHIR StructureDefinitions. GoFSH requires that these files be JSON. Every JSON file contained in the input directory, or its subdirectories, will be processed by GoFSH into FSH.
```

Comment:
GoFSH takes more than SDs.  At the moment, it also processes Code Systems.  Perhaps, "GoFSH takes FHIR StructureDefinitions and other FHIR conformance definitions (e.g., ValueSets, CodeSystems) as input."